packages/paste-types/src/index.ts

```diff
@@ -1,2 +1,9 @@
 export {SendGridThemeShape, ThemeShape} from '@twilio-paste/theme-tokens';
-export * from './tokenProps';
+
+export * from './tokens/backgroundColor';
```

Comment:
Minor nit, these aren't token types, these are style prop types. (Just in case we ever move design token types into this package in the future)